lightning_client.go

```diff
@@ -3093,11 +3093,11 @@ func (p *ChannelClosedUpdate) CloseTxid() chainhash.Hash {
 // sending an EOF), we close the updates and error channel to signal that there
 // are no more updates to be sent. It takes an optional delivery address that
 // funds will be paid out to in the case where we cooperative close a channel
-// that *does not* have an upfront shutdown addresss set.
+// that *does not* have an upfront shutdown address set.
 func (s *lightningClient) CloseChannel(ctx context.Context,
```

Comment:
You'll also need to update the interface definition in `LightningClient`.

Reply:
fixed.